src/webhook/mutation/pod_mutator/oneagent_mutation/versionlabel.go

```diff
@@ -18,11 +26,26 @@ func newVersionLabelMapping(namespace corev1.Namespace) VersionLabelMapping {
 }
 
 func getMappingFromNamespace(namespace corev1.Namespace) VersionLabelMapping {
-	// TODO: Implementation => Parse Namespace annotations into correct format
-	return VersionLabelMapping{}
+	mappingKey2versionLabelKey := map[string]string{
+		versionMappingAnnotationName: releaseVersionEnv,
+		productMappingAnnotationName: releaseProductEnv,
+		stageMappingAnnotationName:   releaseStageEnv,
+		buildVersionAnnotationName:   releaseBuildVersionEnv,
+	}
+
+	versionLabelMapping := VersionLabelMapping{}
+	for mappingKey, versionLabelKey := range mappingKey2versionLabelKey {
+		if fieldRef, ok := namespace.Annotations[mappingKey]; ok {
```

Comment:
```suggestion
		if fieldRef, hasAnnotation := namespace.Annotations[annotation]; hasAnnotation {
```

Reply:
`ok` is idiomatic. I also think it's unfortunate but prefer to follow general rules.

Reply:
if we use golang conventions the line would look like so 
`if f, ok := n.Annotations[a]; ok {`
we already break a lot of conventions because most of them make code unreadable, so if we don't use `ok` for checking if a key is in a map I think thats okay as well

Reply:
It was already discussed here: https://github.com/Dynatrace/dynatrace-operator/pull/981#discussion_r926447779

Reply:
having a special name for `ok` can make sense if its used in a special way, 
but this is a one liner, so having that single line be shorter is preferred

TLDR: `ok` is ok here 🙃